src/plugins/config/Config.tsx

```diff
@@ -77,6 +77,12 @@ const Config: FC = () => {
     <Formik initialValues={initialValues} onSubmit={handleSubmit}>
       <NoPaddingWrapper>
         <ResetForm initialValues={initialValues} />
+        <div>
+          <p style={{marginLeft: '3%', marginTop: '1%', marginBottom: '0px'}}>Variables allow you reuse sections of your config file.
```

Comment:
Please style this using emotion instead of inline styles. as we do with the other components.  

Also probably keep the padding consistent with the toolbar div. https://github.com/Flexget/webui/blob/09898f5fb9e1ec5bd0b9b1255a7fb84a1b9c2b07/src/plugins/config/Config.tsx#L23-L24

Reply:
I did this but I think it looks worse. When I changed to the emotion css stuff, the p tag now has a margin that I can't seem to get rid of it. It doesn't look horrible, but it doesn't look nice either.

Reply:
add `margin: 0` to get rid of that.